src/modules/peek/Peek.FilePreviewer/Previewers/ImagePreviewer/ImagePreviewer.cs

```diff
@@ -168,5 +169,75 @@ private static async Task<BitmapSource> GetBitmapFromHBitmapAsync(IntPtr hbitmap
                 NativeMethods.DeleteObject(hbitmap);
             }
         }
+
+        public static bool IsFileTypeSupported(string fileExt)
+        {
+            return _supportedFileTypes.Contains(fileExt.ToLower());
+        }
+
+        private static readonly string[] _supportedFileTypes = new string[]
```

Comment:
nit: consider making this a hashset for faster lookup times at the cost of memory. @SamChaps am I correct in assuming we want to prioritize performance over memory size?